students/thorn/lesson07/html_render_unit_tests.py

```diff
@@ -0,0 +1,91 @@
+'''
+Thomas Horn
+4.13.2018
+Lesson 7 - HTML Render and Subclassing
+'''
+
+import unittest
+import html_render as hr
+from io import StringIO
+
+# class TagTests(unittest.TestCase):
+#     def test_html(self):
+#         ''' Test that it starts with <html> and ends with </html> '''
+#         e = hr.Element()
+#         test = hr.Html()
+#         e.render(hello)
+#         f = StringIO()
+#         test.render(f)
+#         lines = f.getvalue()
+#         lines.strip()
+#         print(lines)
+
+
+class Step3Tests(unittest.TestCase):
```

Comment:
I would use a single class for all of your test methods in this file. And you can use `unittest.TestCase.setUp` method for shared data and set up logic between methods.